modules/tinymce/CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Notifications no longer require a timeout to disable the close button #TINY-6679
 - The editor theme is now fetched in parallel with the icons, language pack and plugins #TINY-8453
 - Calls to `editor.selection.setRng` now update the cursor position bookmark used when focus is returned to the editor #TINY-8450
+- The default height of editor is changed from `200px` to `400px` #TINY-6860
```

Comment:
As I said you can't use `changed` in an improved entry, so how about something like this:
```suggestion
- The default height of editor has been increased from `200px` to `400px` #TINY-6860
```

Reply:
I am wondering if we need to say why though, as remember this is a UX improvement to make the editor more usable since 200px was deemed too small for content authoring.

Reply:
Going to change it `The default height of editor has been increased from `200px` to `400px` to improve the usability of UI #TINY-6860`?

Reply:
Sounds good 👍

Reply:
actually, maybe say `usability of the editor` rather than the UI?